### DISCUSSION TITLE
关于“加密奇小怪”和明的NFT作品参与HUI的NFT策展计划

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#11] [Ideas]
HUI有一个NFT策展计划，我们有一个小群在讨论这个事情。
目前有两组作品拟参加HUI策展计划：
1，奇小怪logo
奇小怪logo目前有周宇版本和黄同学的“修格奇斯”版本
HUI和MING都建议单纯的奇小怪logo不够，应该向类似《加密猫》和《加密朋克》的方向转化
2，MING的NFT作品
MING已经在opensea开设了NFT画廊，来测试他的很多更酷的构想，我们可以学习MING的NFT画廊的经验

另外，之前的[關於奇小怪的設定以及為它開發一套表情包 ＃3](https://github.com/qihuju/qihuju.github.io/discussions/3)也可以呼应这个策展任务

**Comment by MingxuanXie** [comment]
关于线上策展和NFT的个人看法
1. 组织成为一个“具备玩法的活动”比单纯铸造了几个NFT更重要，这点可能涉及到使用clubhouse开房间来进行赛博逛展等，甚至可以是使基于WebXR技术的三维网页，比如[Mozilla Hubs](https://hubs.mozilla.com/)这种平台。
2. 个人认为NFT交易炒作目前已经过了高峰，单纯的搞NFT艺术拍卖交易的已经太多了甚至有点烂大街。我们不如退一步关注到如何利用区块链的加密机制来设计新颖的“玩法”，在加密猫和加密朋克基础上加入“卡牌”和“共创”等，类似[hashmask](https://www.thehashmasks.com/)，[Waifu](https://nft.waifutoken.io/)等项目，但内容是我们关心的主题。

**Reply by Lupingyue** [reply]
1，非常同意「使用clubhouse开房间来进行赛博逛展等」。「基于WebXR技术的三维网页」目前可以实现了吗？
如果条件成熟，我们随时可以开房间开始测试「赛博逛展」
2，「退一步关注到如何利用区块链的加密机制来设计新颖的“玩法”，在加密猫和加密朋克基础上加入“卡牌”和“共创”等」
……100%同意，NFT不外乎是一种新工具，我们最终关心的是用这个工具创作出什么东西。
三个关键词：玩法、卡牌、共创。估计阿呆会有一些创意出来。
另外，我好像在哪里看过有人已经用NFT做类似“炉石传说”的卡牌游戏了，好像不光是卡牌，也包含了“玩法”

**Reply by Tz-H** [reply]
> 基于WebXR技术的三维网页

see https://immersive-web.github.io/webxr-samples/

**Reply by Tz-H** [reply]
突然发现这个项目也是基于 Github Pages 的，也就是说，我可以直接把这整个项目fork（通俗一点的词就是 copy）到 qihuju 的项目下，通过 qihuju.wiki/webxr 就可以访问

**Reply by Tz-H** [reply]
> 突然发现这个项目也是基于 Github Pages 的，也就是说，我可以直接把这整个项目fork（通俗一点的词就是 copy）到 qihuju 的项目下，通过 qihuju.wiki/webxr 就可以访问

qihuju.wiki/webxr 已经可以访问了 

https://qihuju.wiki/webxr

**Reply by Sleeper2060** [reply]
开房做一个小的warm up挺好的。之前试过用Mozilla和Spatial，通过vr headset和别人的虚拟形象聊天会感觉很有趣。
这样就一点一点的把奇奇们也带到VR体验虚拟世界。👌

**Reply by Lupingyue** [reply]
> 突然發現這個項目也是基於Github Pages的，也就是說，我可以直接把這整個項目fork（通俗一點的詞就是copy）到qihuju的項目下，通過qihuju.wiki/webxr就可以訪問

GitHub真强大，想到什么就能实现的感觉，厉害！

**Reply by Sleeper2060** [reply]
> > 突然發現這個項目也是基於Github Pages的，也就是說，我可以直接把這整個項目fork（通俗一點的詞就是copy）到qihuju的項目下，通過qihuju.wiki/webxr就可以訪問
> 
> GitHub真强大，想到什么就能实现的感觉，厉害！

个人觉得这个虚拟空间的聚会也是可以提上议程了。LOL,期待。。。。。

**Comment by Tz-H** [comment]
我有一个还没有完全思考成熟的Idea，一句话概括就是，结合Clubhouse的语音交流为前置条件，有没有可能实现以语音 -- 文字 -- 图像为路径的一种共生共创艺术作品？

其中语音到文字，文字到图像的转变都是借助当下的计算机算法以及特征提取生成技术（例如生成对抗网络）来实现？

Clubhouse 讲者按照一定的规则和约定讲述一段语音 -- 语音被识别为文字 -- 文字被提取里面的特征 -- 特征被算法生成独特但又上下文互相关联呼应图像装置艺术（怪物卡片？）

**Reply by Sleeper2060** [reply]
脑图可以试试图像的MACHINE LEARNING，出来的影像可以做成gif。

**Reply by Lupingyue** [reply]
> 腦圖可以嘗試圖像的機器學習，出來的影像可以插入gif。

这个要实现估计需要比较复杂的技术吧？

**Reply by Lupingyue** [reply]
> 我有一個還沒有完全思考成熟的想法，一句話概括就是，結合俱樂部俱樂部的語音交流為前提條件，有沒有可能實現以語音-文字-圖像為路徑的一種共生共創藝術作品？
> 
> 其中語音到文字，文字到圖像的轉變都是替換當下的計算機算法以及特徵提取生成技術（例如生成對抗網絡）來實現？
> 
> 會所講師遵循一定的規則和約定的一段語音-語音被識別為文字-文字被提取裡面的特徵-特徵被算法生成獨特但又相互關聯的呼應圖像裝置藝術（怪物卡片？）

这个思路是你一开始就提到的，现在更加具象了。我最感兴趣的是：语音-文字-特征-图像装置艺术-怪物卡牌，这就好像是某种局部的“五感通天塔”

**Reply by Sleeper2060** [reply]
> > 腦圖可以嘗試圖像的機器學習，出來的影像可以插入gif。
> 
> 这个要实现估计需要比较复杂的技术吧？


技术我可以实现，主要可能现在脑图不够多。一般需要200-500张图。
不过我们不是用它来做学术研究，可能10张图也可以得出很好玩的效果。

可以看我网站（[huihuang.space](http://huihuang.space)）首页的佛头像gif，是用machine learning得出来的结果。
我把它放到我的一个3d modeling的空间里面做成了影片。

**Reply by Lupingyue** [reply]
> > > 腦圖可以嘗試圖像的機器學習，出來的影像可以插入gif。
> > 
> > 
> > 这个要实现估计需要比较复杂的技术吧？
> 
> 技术我可以实现，主要可能现在脑图不够多。一般需要200-500张图。
> 不过我们不是用它来做学术研究，可能10张图也可以得出很好玩的效果。
> 
> 可以看我网站（[huihuang.space](http://huihuang.space)）首页的佛头像gif，是用machine learning得出来的结果。
> 我把它放到我的一个3d modeling的空间里面做成了影片。

如果是10张图也能做，现在应该就有十张脑图了，可以试试。

**Reply by Sleeper2060** [reply]
> 如果是10张图也能做，现在应该就有十张脑图了，可以试试。

找时间试试。

**Reply by Lupingyue** [reply]
> > 如果是10张图也能做，现在应该就有十张脑图了，可以试试。
> 
> 找时间试试。

@Sleeper2060 [十张脑图资源牌](https://github.com/qihuju/qihuju.github.io/discussions/21#discussion-3306452)已经准备好了，找时间试试

**Reply by Sleeper2060** [reply]
> > > 如果是10张图也能做，现在应该就有十张脑图了，可以试试。
> > 
> > 
> > 找时间试试。
> 
> @Sleeper2060 [十张脑图资源牌](https://github.com/qihuju/qihuju.github.io/discussions/21#discussion-3306452)已经准备好了，找时间试试

👌

**Reply by Sleeper2060** [reply]
> > > 如果是10张图也能做，现在应该就有十张脑图了，可以试试。
> > 
> > 
> > 找时间试试。
> 
> @Sleeper2060 [十张脑图资源牌](https://github.com/qihuju/qihuju.github.io/discussions/21#discussion-3306452)已经准备好了，找时间试试

脑图好像只有9张？

**Reply by Lupingyue** [reply]
> > @Sleeper2060 [十张脑图资源牌](https://github.com/qihuju/qihuju.github.io/discussions/21#discussion-3306452)已经准备好了，找时间试试
> 
> 脑图好像只有9张？

补上了[第十张](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-587351)

**Reply by Sleeper2060** [reply]
👌

**Comment by MingxuanXie** [comment]
> > 突然发现这个项目也是基于 Github Pages 的，也就是说，我可以直接把这整个项目fork（通俗一点的词就是 copy）到 qihuju 的项目下，通过 qihuju.wiki/webxr 就可以访问
> 
> qihuju.wiki/webxr 已经可以访问了
> 
> https://qihuju.wiki/webxr

要不直接fork这个吧 [peach.land](https://peach.land/) , https://github.com/MingxuanXie/peach.land
这是我最近做的基于 WebXR之上的[Aframe框架](https://aframe.io/) 的基础上，整合一些基础的交互逻辑和对接opensea的api，以及metamask账户验证等。只需要替换内容就可以了。 用Aframe比直接用WebXR方便点。[peach.land](https://peach.land/)已经可以直接用oculus  quest自带浏览器访问并进入VR模式了 @Tz-H

**Reply by Tz-H** [reply]
> > > 突然发现这个项目也是基于 Github Pages 的，也就是说，我可以直接把这整个项目fork（通俗一点的词就是 copy）到 qihuju 的项目下，通过 qihuju.wiki/webxr 就可以访问
> > 
> > 
> > qihuju.wiki/webxr 已经可以访问了
> > https://qihuju.wiki/webxr
> 
> 要不直接fork这个吧 [peach.land](https://peach.land/) , https://github.com/MingxuanXie/peach.land
> 这是我最近做的基于 WebXR之上的[Aframe框架](https://aframe.io/) 的基础上，整合一些基础的交互逻辑和对接opensea的api，以及metamask账户验证等。只需要替换内容就可以了。 用Aframe比直接用WebXR方便点。[Aframe框架](https://aframe.io/) 已经可以直接用oculus quest自带浏览器访问并进入VR模式了 @Tz-H

没问题，你想用啥名字？ qihuju.wiki/peach-land 可以吗？

**Reply by Tz-H** [reply]
@MingxuanXie @Lupingyue 

已经可以访问，地址如下：

https://qihuju.wiki/peach.land/

另：项目名字随时可以调整更新，注意：项目的名字就会是访问的网址名字

**Comment by Sleeper2060** [comment]
个人觉得可以先从比较简单的作品做起。
例如全部奇奇一起做一幅图（用[Exquisite corpse](https://en.wikipedia.org/wiki/Exquisite_corpse#:~:text=Exquisite%20corpse%2C%20also%20known%20as,or%20images%20is%20collectively%20assembled.)的方法），或者把每个人的头像或者名字拼合到奇忽局的图案上。

1，如果要用Exquisite corpse，方法要细化。
一开始的图案可以从奇忽局的图案开始。
通过这个方法出来的图片都蛮有意思的。
[https://www.tate.org.uk/art/art-terms/c/cadavre-exquis-exquisite-corpse](url)


2，如果第二种方式，具体操作可以这样：发起一个活动，每人用1元（作为铸币的费用，9-15刀）可以把自己的名字或头像放到奇忽局的图案上(我可以来操作怎么整合）。铸币的时候也备注每个人的名字，可以作为第一批奇忽局的成员的记录永远在网络存在（如果是用在我的project上，需要我来操作铸币）。这个作品还可以继续流通拍卖，如果还有收入可以作为奇忽局的下一次活动的基金。

**Reply by Lupingyue** [reply]
> 个人觉得可以先从比较简单的作品做起。
> 例如全部奇奇一起做一幅图（用[Exquisite corpse](https://en.wikipedia.org/wiki/Exquisite_corpse#:~:text=Exquisite%20corpse%2C%20also%20known%20as,or%20images%20is%20collectively%20assembled.)的方法），或者把每个人的头像或者名字拼合到奇忽局的图案上。
> 
> 1，如果要用Exquisite corpse，方法要细化。
> 一开始的图案可以从奇忽局的图案开始。
> 通过这个方法出来的图片都蛮有意思的。
> [https://www.tate.org.uk/art/art-terms/c/cadavre-exquis-exquisite-corpse](url)
> 
> 2，如果第二种方式，具体操作可以这样：发起一个活动，每人用1元（作为铸币的费用，9-15刀）可以把自己的名字或头像放到奇忽局的图案上(我可以来操作怎么整合）。铸币的时候也备注每个人的名字，可以作为第一批奇忽局的成员的记录永远在网络存在（如果是用在我的project上，需要我来操作铸币）。这个作品还可以继续流通拍卖，如果还有收入可以作为奇忽局的下一次活动的基金。

好棒的创意，琢磨琢磨。

**Reply by Sleeper2060** [reply]
如果觉得OK，我们四个人可以决定其中一种方式，由我来细化方案。
然后大家再给意见整合优化，这样可以推动project的前进。
期间也可以不断加新的主意或者内容进来。

**Reply by Lupingyue** [reply]
> 如果觉得OK，我们四个人可以决定其中一种方式，由我来细化方案。
> 然后大家再给意见整合优化，这样可以推动project的前进。
> 期间也可以不断加新的主意或者内容进来。

我觉得可以啊，看看 @Tz-H 和 @MingxuanXie 的意见。

**Comment by Tz-H** [comment]
在知乎上看到一个相关的话题，其中引用的技术和网站很适合作为我们正在讨论的策展方案的一个参考原型。

知乎参考问题：人工AI绘画是否会让中低端画师失业？
链接：[https://www.zhihu.com/question/442610947/answer/1789104906](https://www.zhihu.com/question/442610947/answer/1789104906)

技术描述：

> 推荐一个免费网站 Artbreeder，可通过合成任意数量的图片，产生新的子集作品，并可通过调节参数改变效果。
> 此网站还有制作风景、生成动图等多种功能，大家可自行探索。


引用的技术网站：Artbreeder
链接：[https://www.artbreeder.com/create](https://www.artbreeder.com/create)

参考效果：

![image](https://user-images.githubusercontent.com/1928161/113418962-64d9d980-93be-11eb-93e2-b6554d8e3317.png)

![image](https://user-images.githubusercontent.com/1928161/113418970-6905f700-93be-11eb-90b0-153c93d4974a.png)

![image](https://user-images.githubusercontent.com/1928161/113418985-6efbd800-93be-11eb-805b-a9ff4d10fd5f.png)

动图案例：

[点击查看](https://artbreeder.b-cdn.net/static/video/animate/portraits_sg2.mp4)

--------

一个额外引申的思考点：
> 要是能把这个Artbreeder的算法和unreal捏脸工具结合起来，就能随机生成带有父代遗传信息的3D小人了

**Reply by Lupingyue** [reply]
这个工具很棒，不过这个针对“具象”的形象才有效。目前奇小怪的logo是抽象的，所以用不到这个工具。这就需要一个“具象”的新作品「比如HUI前面提到的『把每个人的头像或者名字拼合到奇忽局的图案上』，就可以是年轻版的每个人的头像or年老版『2200年』的每个人的头像」来激活这个工具。

**Reply by Sleeper2060** [reply]
嗯嗯，一段融合所有人的头像的影像也是蛮不错的哦。

**Reply by Lupingyue** [reply]
> 嗯嗯，一段融合所有人的头像的影像也是蛮不错的哦。

![1553566451876](https://user-images.githubusercontent.com/25149468/113427000-d8073e00-9406-11eb-92ef-a6a7d8f3a1ed.jpeg)
细化一下这个方案：把每个人的头像或者名字拼合到奇忽局的logo图案上
1，“每个人”应该是自愿的，至少我们应该有4个人的头像，保证这个计划不失败
2，另外，我们可以加上21个「2021年的意思」世界名人未来某年的头像「2200，2045，2041什么的都可以，找到一个梗就可以」，类似“甲壳虫”乐队专辑《Sgt. Pepper's Lonely Hearts Club Band》封面的玩法：甲壳虫乐队+世界名人的人像

**Reply by Sleeper2060** [reply]
haha,票选21位名人加入也是非常有趣的。👏

像你发的collage带点幽默感也是蛮好玩的。
我原来想的是photo mosaic的感觉，
[![](https://thepeoplespicture.com/wp-content/uploads/2014/10/ftproject01.jpg)](url)

在中间组成奇忽局logo的图案。

**Reply by Lupingyue** [reply]
> haha,票选21位名人加入也是非常有趣的。👏
> 
> 
> 
> 像你发的collage带点幽默感也是蛮好玩的。
> 
> 我原来想的是photo mosaic的感觉，
> 
> [![](https://thepeoplespicture.com/wp-content/uploads/2014/10/ftproject01.jpg)](url)
> 
> 
> 
> 在中间组成奇忽局logo的图案。

图案不一定是甲壳虫这个封面，我其实主要想说明：真人+名人的组合。
我觉得目前比较有趣的是两点：
1，多人头像构成奇小怪logo图案
2，这些头像的图案是使用 @Tz-H 推荐的这个捏脸工具“捏造”的“未来脸”

**Comment by Sleeper2060** [comment]
方案已经发布在PROJECT SECTION. 
 #希望大家浏览提出意见与建议。🙏
[https://github.com/qihuju/qihuju.github.io/projects/2](url)

**Reply by Lupingyue** [reply]
> 方案已经发布在PROJECT SECTION.
> #希望大家浏览提出意见与建议。🙏
> [https://github.com/qihuju/qihuju.github.io/projects/2](url)

棒呆。
建议：是否成立一个“奇小怪NFT化理事会”，或者更宽泛一点，“NFT策展理事会”？

**Reply by Sleeper2060** [reply]
或者叫NFTs艺术文化事务理事会
😄

**Reply by Lupingyue** [reply]
> 或者叫NFTs艺术文化事务理事会
> 😄

可以啊，你决定。
